src/components/wizard/WizardStep.js

```diff
@@ -18,6 +18,24 @@ import {States, StatesConfig} from './WizardStates';
  *
  * @example: https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/WizardScreen.js
  * @notes: Use Wizard with nested Wizard.Step(s) to achieve the desired result.
+ * @configuration: const configMap = {<br/>
```

Comment:
keep it under `@notes`. 
no need for a new doc field

Reply:
I'd like to have the code indented properly (using `<code><code/>` and `code: {white-space: pre;}`).